### PR TITLE
docs: fix footer style

### PR DESCRIPTION
**Pull request by Anna-MariiaPetryk**
# PR Details

Footer component bugfixes

### CODE REVIEW

File: pages/views/_includes/footer.njk [hunk 1]
```diff
@@ -25,7 +25,14 @@
...
   </esl-panel-group>
   <div class="footer-dropdown footer-links">
-    <a href="#" class="footer-link">Terms & Conditions</a>
-    <a href="#" class="footer-link">Privacy Policy</a>
+    <div class="footer-link">
+      <a href="#">Terms & Conditions</a>
+      <a href="#">Privacy Policy</a>
+    </div>
+    <div class="footer-github footer-link">
+      <a href="{{ site.github.url }}" target="_blank">
```
Comment (ala-n) [inline pages/views/_includes/footer.njk:33]: `class` can be added to a tag

File: pages/views/_includes/footer.njk [hunk 2]
```diff
@@ -25,7 +25,14 @@
     </ul>
   </esl-panel-group>
   <div class="footer-dropdown footer-links">
-    <a href="#" class="footer-link">Terms & Conditions</a>
-    <a href="#" class="footer-link">Privacy Policy</a>
+    <div class="footer-link">
+      <a href="#">Terms & Conditions</a>
+      <a href="#">Privacy Policy</a>
+    </div>
```
Comment (ala-n) [inline pages/views/_includes/footer.njk:31]: something wrong here
- no classes on anchors
- single footer-link for two anchors

File: pages/src/footer/footer.less [hunk 3]
```diff
@@ -93,12 +92,19 @@
...
+    display: flex;
+    justify-content: space-between;
     padding-bottom: 20px;
   }
 
   &-dropdown-container {
     margin: 0;
   }
+
+  &-github{
```
Comment (ala-n) [inline pages/src/footer/footer.less:104]: Can be abstract '&-social-link' or '&-icon-link'

**Review by ala-n — CHANGES_REQUESTED** [review]

File: pages/src/footer/footer.less [hunk 4]
```diff
@@ -32,8 +36,8 @@
     justify-content: center;
     cursor: pointer;
     margin-left: 30px;
-    width: 150px;
-    height: 35px;
+    width: 180px;
+    height: 55  px;
```
Comment (ala-n) [inline pages/src/footer/footer.less:40]:
```suggestion
    height: 55px;
```
Extra spaces, that may cause an issues

File: pages/src/footer/footer.less [hunk 5]
```diff
@@ -46,38 +50,74 @@
...
+  &-accordion-trigger {
     display: flex;
     align-items: center;
     justify-content: center;
     height: 46px;
   }
 
-  &-dropdown-title {
-    font-weight: 500;
+  &-accordion-trigger[aria-expanded="true"]{
```
Comment (ala-n) [inline pages/src/footer/footer.less:108]: Do not use aria attribute in the styles!!
Aria attributes only consumes the state of component but they should not deliver it

File: pages/views/_includes/footer-menu.njk [hunk 6]
```diff
@@ -1,19 +1,19 @@
 {% macro menu(title, collection) %}
-  <esl-trigger class="footer-dropdown-trigger"
+  <esl-trigger class="footer-accordion-trigger"
                target="::next"
                hover-hide-delay="400">
-    <span class="footer-dropdown-title {{ 'nav-item-selected' if collection in page.url }}" href="#">
+    <div class="footer-accordion-title footer-accordion-arrow {{ 'nav-item-selected' if collection in page.url }}" href="#">
```
Comment (ala-n) [inline pages/views/_includes/footer-menu.njk:5]: `href` is not presented on the div tag

**Review by ala-n — CHANGES_REQUESTED** [review]

**Review by ala-n — APPROVED** [review]

**Review by dshovchko — APPROVED** [review]

**Review by abarmina — APPROVED** [review]

**Pull request merged**